quail/src/main/java/com/mineinjava/quail/pathing/Path.java

```diff
@@ -34,26 +34,26 @@ public Path(ArrayList<Pose2d> points) {
     /** returns the next point in the path. Also increments the current point.
      */
     public Pose2d getNextPoint() {
-        if (currentPoint < lastpoint) {
-            return points.get(currentPoint + 1);
+        if (currentPointIndex < lastpoint) {
+            return points.get(currentPointIndex + 1);
         } else {
             return null;
         }
     }
     /** returns the current point in the path.
      */
-    public Pose2d getCurrentPoint() {
-        if (currentPoint <= lastpoint) {
-            return points.get(currentPoint);
+    public Pose2d getCurrentPointIndex() {
```

Comment:
this shouldn't be `getCurrentPointIndex` as it returns a pose2d object and not an integer